src/scripts/main.js

```diff
@@ -1,3 +1,37 @@
 'use strict';
 
-// write code here
+const liElements = document.querySelectorAll('li');
+
+function salaryToNumber(string) {
+  return +string.slice(1).replaceAll(',', '');
+}
+
+function getEmployees(list) {
+  const persons = [];
+
+  for (const item of list) {
+    persons.push({
+      name: item.textContent.trim(),
+      position: item.dataset.position,
+      salary: item.dataset.salary,
+      age: item.dataset.age,
```

Comment:
You can use destructuring for the item.dataset and simplify this code 
```suggestion
      position,
      salary,
      age,
```